test/helpers.test.js

```diff
@@ -63,6 +64,30 @@ describe('helpers', function() {
     });
   });
 
+  // test checkRemoteMethodName()
+  describe('checkRemoteMethodName()', function() {
+    it('should accept good names', function() {
+      testValidationAcceptsValue(checkRemoteMethodName, 'prop');
+      testValidationAcceptsValue(checkRemoteMethodName, 'prop1');
+      testValidationAcceptsValue(checkRemoteMethodName, 'my_prop');
+      testValidationAcceptsValue(checkRemoteMethodName, 'my-prop');
+      testValidationAcceptsValue(checkRemoteMethodName, 'm.prop');
```

Comment:
Please use `prototype.method` to make it clearer that we want to support this particular case.

I think we should not allow `.` in method names except the `/^prototype\./` case, i.e. `m.prop` is not a valid method name.